lib/ubus.c

```diff
@@ -819,6 +824,7 @@ uc_ubus_call(uc_vm_t *vm, size_t nargs)
 	               "method", UC_STRING, REQUIRED, &funname,
 	               "data", UC_OBJECT, OPTIONAL, &funargs,
 	               "multiple_return", UC_BOOLEAN, OPTIONAL, &mret,
```

Comment:
I wonder if it makes sense to introduce a new option `return` here which takes a number of specific string values:
- `"single"` (the default, one result)
- `"multiple"` (supersedes mret)
- `"ignore"` (no return, immediately abort after call)